cumulus/tasks/discover-granules/tests/test.js

```diff
@@ -1,77 +1,80 @@
 'use strict';
 
 const test = require('ava');
-const sinon = require('sinon');
-const log = require('@cumulus/common/log');
-const amsr2 = require('@cumulus/test-data/payloads/amsr2/discover.json');
-const queue = require('@cumulus/ingest/queue');
 const mur = require('./fixtures/mur.json');
-const { handler } = require('../index');
+const { cloneDeep } = require('lodash');
```

Comment:
missing from `package.json`. I suggest we use `lodash.cloneDeep` instead of the `lodash`